assets/constants.ts

```diff
@@ -417,6 +417,7 @@ export default {
   "STORY_NODE_STATUS_PUBLISHED": "Published",
   "STORY_NODE_STATUS_READY_TO_PUBLISH": "Ready To Publish",
   "STORY_NODE_STATUS_DRAFT": "Draft",
+  "UPCOMING_CHAPTERS_DAY_LIMIT": 14,
```

Comment:
Basically, for considering a chapter as upcoming, the planned publication date should be within a limit of 14 days. But in this or some other PR earlier, I was asked to remove 14 and replace it with a constant.. Considering that, I hope now the meaning of the constant  seems to satisfy the purpose maybe? Or if you can suggest an alternative for it?

Reply:
Part of what a developer needs to do is to figure out good names for the concepts being represented, so, next time, please try to make some suggestions as well, rather than just request that one be given directly.

I thought about it and I think the idea here is that of a "notice period". So perhaps something like CHAPTER_PUBLICATION_NOTICE_PERIOD_IN_DAYS or similar.

Reply:
Done